src/routes/volume/detail/VolumeInfo.js

```diff
@@ -282,9 +295,11 @@ function VolumeInfo({ selectedVolume, snapshotModalState, engineImages, hosts, c
         </div>
       </div> : ''}
       <div className={styles.row}>
-        <Tooltip title={'Provides the binary to start and communicate with the volume engine/replicas.'}>
-          <span className={styles.label}> Engine Image:</span>
-        </Tooltip>
+        <span className={styles.label}>
+          <span>Engine Image</span>
+          {renderLabelTooltip('Provides the binary to start and communicate with the volume engine/replicas.')}
+          <span>:</span>
+        </span>
```

Comment:
In addition to `Size` and `Actual Size`, align the same mechanism with `Engine Image` and `Instance Manager`.

Reply:
> In addition to `Size` and `Actual Size`, align the same mechanism with `Engine Image` and `Instance Manager`.

Does https://github.com/longhorn/longhorn/issues/8843 required backport to v1.7? I didn't see any backport label attached. cc @derekbit